esp-hal-common/src/interrupt/riscv.rs

```diff
@@ -21,6 +21,7 @@ pub use self::classic::*;
 pub use self::plic::*;
 use crate::{
     peripherals::{self, Interrupt},
+    riscv,
```

Comment:
when the feature is not enabled this will give a warning about unused imports

Reply:
Thanks, i've moved this into the helpers now.